Bottom: 84dde5925d2ae4b35a6fc92a878e89811540430a
Top:    d551dadf671871078fc6a8314f4015e8e5d6e78a
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-20 22:44:34 -0700

Refresh of destroy_cqs

---

diff --git a/src/rsocket.c b/src/rsocket.c
index de0ec3e..01b7248 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -366,11 +366,8 @@ static void rs_free(struct rsocket *rs)
 		rdma_dereg_mr(rs->target_mr);
 
 	if (rs->cm_id) {
-		if (rs->cm_id->qp) {
-			rs->cm_id->send_cq_channel = NULL;
-			rs->cm_id->send_cq = NULL;
+		if (rs->cm_id->qp)
 			rdma_destroy_qp(rs->cm_id);
-		}
 		rdma_destroy_id(rs->cm_id);
 	}
